src/services/workflow.service.js

```diff
@@ -354,7 +356,6 @@ class WorkflowService {
         )
         this.subscriptions[subscription.query.name] = subscription
         // All done!
-        subscription.handleViewState(ViewState.COMPLETE, null)
```

Comment:
Moving this has broken some unit tests.

Reply:
Good!

Will fix them to the new behaviour.